typescript/utils/src/altvm.ts

```diff
@@ -0,0 +1,657 @@
+import { MINIMUM_GAS } from './mingas.js';
+import { ProtocolType } from './types.js';
+
+// ### QUERY BASE ###
+export type ReqGetBalance = { address: string; denom: string };
```

Comment:
nit: I would prefer if instead of following the naming convention `Req<action><resource>` we use `<action><resource><context>`. Example:
- ReqGetBalance -> GetBalanceOptions
- ResGetBalance -> GetBalanceResponse

it makes it easier to read to me

Reply:
For "this one" I meant that multiple renaming changes where requested which I all implemented except for "this one". I would also see this in the upcoming multi-vm design doc discussion about the final interface since this is no pressing issue or bug